utils/runtime/chaos-result.j2

```diff
@@ -2,13 +2,17 @@
 apiVersion: litmuschaos.io/v1alpha1
 kind: ChaosResult
 metadata:
-  {% if instance_id is defined %}
+  {% if instance_id is defined and c_result is defined %}
   name: {{ c_result }}-{{ instance_id }}
-  {% else %}
+  {% elif c_result is defined %}
   name: {{ c_result }}
+  {% else %}
+  name: {{ c_experiment }}
   {% endif %} 
   labels:
+    {% if chaos_uid is defined %}
     chaosUID: {{ chaos_uid }}
+    {% endif %} 
```

Comment:
Please put the labels: after the if conditions for now. Otherwise it will failed to create the chaosresult. In future if new labels will add this can be outside of the if conditions

Reply:
Added `name=chaosResult` label for this. It is good to have a label

Reply:
Otherwise you can generate and random string for chaosUID: {{ chaos_uid }}

Reply:
If we run an experiment via job then there is no point of creating chaos_uid label. It is only needed for the cleanup purpose when we run exp via chaos_engine.